clap_derive/src/item.rs

```diff
@@ -70,6 +70,66 @@ impl Item {
         Ok(res)
     }
 
+    pub(crate) fn from_args_enum_variant(
```

Comment:
this method is basically a copy of `from_subcommand_enum_variant`.
I'm not yet entirely clear if that is sensible since my picture of what `Item` is is still somewhat blurry.

Reply:
Good to know, is there any better documentation on `Item` that I can read up on?

Reply:
Not really.  A lot of this is ad hoc.  It'd be good to clean up at some point.